### PR TITLE
Calcite 3

### PR DESCRIPTION
**Pull request by jmhauck**
This moves the location of the demos as well...will now be exposed via `www` folder vs `dist` folder

### CODE REVIEW

File: src/components/layer-table/layer-table.tsx [hunk 1]
```diff
@@ -1906,7 +1906,7 @@ export class LayerTable {
    * @param number[] the selected ids
```
Comment (MikeTschudi) [inline src/components/layer-table/layer-table.tsx:1906]: Shouldn't this be `@returns`?

File: src/components/layer-table/layer-table.tsx [hunk 2]
```diff
@@ -1906,7 +1906,7 @@ export class LayerTable {
...
    */
   protected _getIds(): number[] {
-    return this._table?.highlightIds?.toArray();
+    return this._table?.highlightIds?.map(id => parseInt(id.toString(), 10)).toArray();
```
Comment (MikeTschudi) [inline src/components/layer-table/layer-table.tsx:1909]: What is the value to converting an id to a string and then parsing it back to a number?

File: src/components/spatial-ref/test/spatial-ref.e2e.ts [hunk 3]
```diff
@@ -36,7 +36,7 @@ describe('spatial-ref', () => {
     expect(await spatial_ref.getProperty('value')).toBe('102100');
   });
 
-  it('select first value after typing in search field', async () => {
+  xit('select first value after typing in search field', async () => {
```
Comment (MikeTschudi) [inline src/components/spatial-ref/test/spatial-ref.e2e.ts:39]: Should this have gone back to `it`?

File: src/demos/map-draw-tools.html [hunk 4]
```diff
@@ -42,13 +42,19 @@
     }
   </style>
 
-  <link rel="stylesheet" href="https://jsdev.arcgis.com/4.31/esri/themes/light/main.css" />
+  <script src="https://jsdev.arcgis.com/4.31/"></script>
```
Comment (MikeTschudi) [inline src/demos/map-draw-tools.html:45]: Should the 4.31 version be here?

File: src/demos/map-select-tools.html [hunk 5]
```diff
@@ -45,13 +45,19 @@
     }
   </style>
 
-  <link rel="stylesheet" href="https://jsdev.arcgis.com/4.31/esri/themes/light/main.css" />
+  <script src="https://jsdev.arcgis.com/4.31/"></script>
```
Comment (MikeTschudi) [inline src/demos/map-select-tools.html:48]: Another 4.31?

File: src/utils/makeExternal.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import type { Plugin, ResolveIdResult } from 'rollup';
```
Comment (MikeTschudi) [inline src/utils/makeExternal.ts:1]: Should `makeExternal` and `plugin` have copyright statements?

File: src/components/layer-table/layer-table.tsx (hunk 2 above)
Comment (jmhauck) [reply src/components/layer-table/layer-table.tsx:1909]: @MikeTschudi I'm open to a better way if you know one. It's a for the linting...highlight ids are a `number[] | string[]` so this just makes sure it ends up a number[]

File: src/components/spatial-ref/test/spatial-ref.e2e.ts (hunk 3 above)
Comment (jmhauck) [reply src/components/spatial-ref/test/spatial-ref.e2e.ts:39]: @MikeTschudi The piercing selector `>>>` doesn't seem to work. I went circles with it for awhile.

File: src/demos/map-draw-tools.html (hunk 4 above)
Comment (jmhauck) [reply src/demos/map-draw-tools.html:45]: good eye

File: src/components/spatial-ref/test/spatial-ref.e2e.ts (hunk 3 above)
Comment (jmhauck) [reply src/components/spatial-ref/test/spatial-ref.e2e.ts:39]: I plan to devote an entire day at least to adding some basic tests for this project soon.

**Pull request merged**